### PR TITLE
Fixed #30648 -- Fixed documentation

### PR DESCRIPTION
**Pull request by gachdavit**
Overriding get_context_data() is unnecessary in the "Using FormMixin with DetailView" example.

Ticket Visit https://code.djangoproject.com/ticket/30648.

### CODE REVIEW

**Comment by felixxm** [comment]
Please remove `get_context_data()` also from the above example.

File: docs/topics/class-based-views/mixins.txt [hunk 1]
```diff
@@ -324,11 +324,6 @@ Now we can write a new ``PublisherDetail``::
             self.object = self.get_object(queryset=Publisher.objects.all())
             return super().get(request, *args, **kwargs)
 
-        def get_context_data(self, **kwargs):
-            context = super().get_context_data(**kwargs)
-            context['publisher'] = self.object
-            return context
-
```
Comment (felixxm) [inline docs/topics/class-based-views/mixins.txt]: Please revert this change, you should remove `get_context_data()` only from the first example in the _"Using FormMixin with DetailView"_ paragraph.

Comment (gachdavit) [reply docs/topics/class-based-views/mixins.txt]: Unfortunately, I had a lot of mistakes on the first try,,,

I think, it's ready now.

**Comment by felixxm** [comment]
@gachdavit Thanks for this patch :+1: Welcome aboard :boat:
